fdbserver/QuietDatabase.actor.cpp

```diff
@@ -284,6 +284,52 @@ ACTOR Future<vector<WorkerInterface>> getStorageWorkers(Database cx,
 	return result;
 }
 
+int64_t extractMaxQueueSize(const std::vector<Future<TraceEventFields>>& messages,
```

Comment:
Needs a comment.  The name sounds rather self explanatory but accepting futures as input and requiring them to be already ready is probably worth mentioning.

Reply:
Agree, this was an oversight -- I did this refactoring while debugging the code so it was made in a hurry. I added a comment.